testing/scenario_app/ios/Scenarios/Scenarios/AppDelegate.m

```diff
@@ -0,0 +1,14 @@
+#include "AppDelegate.h"
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication*)application
+    didFinishLaunchingWithOptions:(NSDictionary*)launchOptions {
+  self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
+  self.window.rootViewController = [[FlutterViewController alloc] init];
```

Comment:
oh, can we not check in the storyboard below then?

Reply:
We could, I just wanted a simple app that didn't involve storyboards.

Reply:
Ahh I see, I'll remove the file